src/main/java/org/jitsi/videobridge/simulcast/SimulcastReceiver.java

```diff
@@ -562,6 +571,10 @@ else if (simStream == acceptedStream)
         }
 
 
```

Comment:
Ideally we should split this method because it is way too long, but that's not related to the PR. It isn't obvious that changedStreams doesn't contain duplicates, and we probably want it to not have duplicates. I think we should use a Set here (after fixing SimulcastStream#hashCode)